docs/index.md

```diff
@@ -0,0 +1,57 @@
+# dbt-loom
+
+`dbt-loom` is a dbt Core plugin that weaves together multi-project deployments. It works by fetching public model definitions from your dbt artifacts, and injecting those models into your dbt project.
+
+```mermaid
+flowchart LR
+
+   subgraph TOP[Your Infrastructure]
+    direction TB
+    dbt_runtime[dbt Core]
+    proprietary_plugin[Open Source Metadata Plugin]
+
+    files[Local and Remote Files]
+    object_storage[Object Storage]
+    discovery_api[dbt Cloud APIs]
+
+    discovery_api --> proprietary_plugin
+    files --> proprietary_plugin
+    object_storage --> proprietary_plugin
+    proprietary_plugin --> dbt_runtime
+  end
+
+  Project --> TOP --> Warehouse
+```
+
+
+dbt-loom currently supports obtaining model definitions from:
+
+- Local manifest files
+- Remote manifest files via http(s)
+- dbt Cloud
+- GCS
+- S3-compatible object storage services
+- Azure Storage
+
+!!! warning 
+    dbt core's plugin functionality is still in beta. Please note that this may break in the future as dbt Labs solidifies the dbt plugin API in future versions.
```

Comment:
**Suggestion:** I think the plugin interface has been around long enough that we're good to remove this warning.